src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -10939,26 +10944,52 @@ def _window_agg(
         # Handle case where min_periods = None
         min_periods = 0 if min_periods is None else min_periods
         # Perform Aggregation over the window_expr
-        new_frame = frame.update_snowflake_quoted_identifiers_with_expressions(
-            {
-                # If aggregation is count use count on row_position_quoted_identifier
-                # to include NULL values for min_periods comparison
-                quoted_identifier: iff(
-                    count(col(row_position_quoted_identifier)).over(window_expr)
-                    >= min_periods
-                    if agg_func == "count"
-                    else count(col(quoted_identifier)).over(window_expr) >= min_periods,
-                    get_snowflake_agg_func(agg_func, agg_kwargs)(
-                        # Expanding is cumulative so replace NULL with 0 for sum aggregation
-                        builtin("zeroifnull")(col(quoted_identifier))
-                        if window_func == WindowFunction.EXPANDING and agg_func == "sum"
-                        else col(quoted_identifier)
-                    ).over(window_expr),
-                    pandas_lit(None),
-                )
-                for quoted_identifier in frame.data_column_snowflake_quoted_identifiers
-            }
-        ).frame
+        if agg_func == "sem":
+            # Standard error of mean (SEM) does not have native Snowflake engine support
+            # so calculate as STDDEV/SQRT(N-1)
```

Comment:
The divisor for sem is N - ddof right, are you only supporting ddof=1 here then?

Reply:
Oh good point adding support/tests for this